### PR TITLE
integrate srtool

### PR DESCRIPTION
**Pull request by GopherJ**
close: #222 
close: #185 
close: #195 

download wasm here: https://github.com/parallel-finance/parallel/actions/runs/952074365

- [ ] test if the wasm works with collator built locally

### CODE REVIEW

**Comment by GopherJ** [comment]
image

the two runtime have the same `hash` which isn't normal, I think maybe we need to finish #181 to seperate these two runtime instead of building one with both `heiko` and `parallel`

**Review by ayushmishra2005 — APPROVED** [review]

File: node/parallel/src/client.rs [hunk 1]
```diff
@@ -176,7 +180,9 @@ impl ClientHandle for Client {
 impl sc_client_api::UsageProvider<Block> for Client {
     fn usage_info(&self) -> sc_client_api::ClientInfo<Block> {
         match self {
+            #[cfg(feature = "with-parallel-runtime")]
```
Comment (kaichaosun) [inline node/parallel/src/client.rs:183]: I remember we discussed that won't increase code complexity with these features for build speedup.

Comment (GopherJ) [reply node/parallel/src/client.rs:183]: looks like the only difference is `WithdrawAmount`  config

Comment (GopherJ) [reply node/parallel/src/client.rs]: I reverted that commit for now

**Pull request merged**